test/regress/383.test

```diff
@@ -23,7 +23,7 @@ test bal Assets:Investments --lots --date-format %Y-%m-%d
 100 AA {2.00 GBP} [2014-01-01]  Assets:Investments
 end test
 
-test bal Assets:Investments --lots --date-format %g-%b-%d
+test bal Assets:Investments --lots --date-format %y-%b-%d
```

Comment:
%g is not supported by [Windows strftime](https://msdn.microsoft.com/en-us/library/fe06s4ak.aspx), so I switched to %y.